sweep/aggregator.go

```diff
@@ -160,7 +144,7 @@ func (c *inputCluster) CreateInputSets(
 // sweeping transaction.
 type UtxoAggregator interface {
 	// ClusterInputs takes a list of inputs and groups them into clusters.
-	ClusterInputs(pendingInputs) []Cluster
+	ClusterInputs(Wallet, pendingInputs) []InputSet
```

Comment:
Similar question here re necessity of accepting the wallet.

Reply:
yeah this look weird - it's done so it's more clear what the next commit is about, that it tries to decrease the coin lock scope, and the wallet is not passed to `ClusterInputs` anymore.